src/packages/cli/src/Init.ts

```diff
@@ -76,7 +94,7 @@ export const defaultURL = (
     case 'sqlserver':
       return `sqlserver://localhost:${port};database=mydb;user=SA;password=randompassword;`
     case 'mongodb':
-      return `mongodb://johndoe:randompassword@localhost:${port}/mydb?authSource=admin`
+      return `mongodb+srv://root:randompassword@cluster0.ab1cd.mongodb.net/mydb?retryWrites=true&w=majority`
```

Comment:
Is this intended?

Reply:
I copy paste from example in the (not yet merged) docs, I guess it's what we want.